Automattic/sensei
# Update stylings of quiz questions in the submitted state in learning mode

Resolves https://github.com/Automattic/sensei/issues/7128

Stacked on https://github.com/Automattic/sensei/pull/7124

## Proposed Changes
* Update stylings of quiz questions in the submitted state in learning mode
* Also fixes the checkbox appearing broken when checked and no multi-line question is there in the quiz

## Testing Instructions


1. Check out this branch of Course theme https://github.com/Automattic/themes/pull/7331
2. Check out this branch of Pro https://github.com/Automattic/sensei-pro/pull/2404
3. Enable course theme
4. Create a course with lessons
5. Have at least one lesson with a quiz that has all the questions in it
6. Make sure Learning Mode is enabled for that course
7. Now answer the quiz questions and click "Quiz Complete"
8. Check in mobile screen size as well
9. Change the theme variation to gold, dark and blue and check the styles for them too.
10. Now switch to another block theme and check
11. Switch to a non-block theme and check
12. For all themes and variations, also check with LM disabled to make sure non-LM style isn't broken

#### Desktop
![image](https://github.com/Automattic/sensei/assets/6820724/23df3bb8-87ed-411b-a53b-3bb05200e750)

#### Mobile
![image](https://github.com/Automattic/sensei/assets/6820724/bc5df021-dca4-47b8-a242-3cc4e7aa2d88)

## New/Updated Hooks
`sensei_quiz_question_answers_inside_before` - Hook used for printing content inside the question answers section before the template content.

`sensei_quiz_question_answers_inside_after` - Hook used for printing content inside the question answers section after the template content.

## Pre-Merge Checklist
